### PR TITLE
Fat 2680 c350678

### PR DESCRIPTION
**Pull request by TetianaParanich**
***
https://issues.folio.org/browse/FAT-2680
***
https://foliotest.testrail.io/index.php?/cases/view/350678
***

### CODE REVIEW

File: cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js [hunk 1]
```diff
@@ -38,6 +38,11 @@ const actions = {
   findAndRemoveThese: 'Find and remove these'
```
Comment (IhorBohdan) [inline cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js:38]: Fix linter issues

File: cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js [hunk 2]
```diff
@@ -400,5 +387,36 @@ export default {
...
+      Select({ name:'profile.mappingDetails.marcMappingDetails[0].field.subfields[0].subaction' }).choose(subaction),
+      TextArea({ name:'profile.mappingDetails.marcMappingDetails[0].field.subfields[0].data.text' }).fillIn(subfieldTextInFirstField),
+      TextField({ name:'profile.mappingDetails.marcMappingDetails[0].field.subfields[1].subfield' }).fillIn(subfieldInSecondField),
+      TextArea({ name:'profile.mappingDetails.marcMappingDetails[0].field.subfields[1].data.text' }).fillIn(subfieldTextInSecondField)
+    ]);
+  },
+
+  fillModificationSectionWithDelete:(action, fieldNumber) => {
+    cy.do([
+      Select({ name:'profile.mappingDetails.marcMappingDetails[0].action' }).choose(action),
```
Comment (IhorBohdan) [inline cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js:410]: move to const

File: cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js [hunk 3]
```diff
@@ -400,5 +387,36 @@ export default {
...
+      TextField({ name:'profile.mappingDetails.marcMappingDetails[0].field.field' }).fillIn(fieldNumber)
```
Comment (IhorBohdan) [inline cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js:411]: move to const

File: cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js (hunk 1 above)
Comment (TetianaParanich) [reply cypress/support/fragments/data_import/mapping_profiles/newFieldMappingProfile.js:38]: Done

**Review by IhorBohdan — APPROVED** [review]

**Review by MaksimTerekhov — APPROVED** [review]

**Pull request merged**